kuma/wiki/models.py

```diff
@@ -1739,6 +1739,9 @@ def content_cleaned(self):
 
     @cached_property
     def previous(self):
+        return self._previous()
+
+    def _previous(self):
```

Comment:
I removed the private method but kept the regular syntax of the cached_property b/c it's more "scannable" to me.